File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 7/IWillKissVaNal.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_7
{
	internal class IWillKissVaNal : StoryChoice
	{
		internal IWillKissVaNal() : base(StoryChoice.GetLocalizedString("ch7IWillKissVaNal"), 7071)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch7KissedVaNal = VaNalKiss.Kissed;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_7/IWillKissVaNal.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_7;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.VaNalKiss;

/**
 * Created by christopherraleigh on 2016-05-19.
 */
public class IWillKissVaNal extends StoryChoice {
    public IWillKissVaNal() {
        super(R.string.ch7IWillKissVaNal, 7071);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh7KissedVaNal(VaNalKiss.Kissed);
        super.select();
    }
}
